Dockerfile

```diff
@@ -1,7 +1,7 @@
 # see https://pythonspeed.com/articles/alpine-docker-python/ for the reason we don't use alpine
 FROM python:3.9-slim
 RUN apt-get update \
-    && apt-get install -y --no-install-recommends git ssh socat wget curl libcairo2 python3-dev libffi-dev socat \
+    && apt-get install -y --no-install-recommends git ssh socat wget curl gcc libcairo2 python3-dev libffi-dev socat \
```

Comment:
Can we use a multi-stage Dockerfile so that we can exclude gcc from the final image? (gcc has a bunch of dependencies so the final image will be smaller this way)

https://docs.docker.com/develop/develop-images/multistage-build/#use-multi-stage-builds